api/app/models/persona.py

```diff
@@ -4,22 +4,34 @@
 from sqlmodel import Field, Column, String, Relationship
 
 from app.models.base import Base
+from app.models.organization import Organization
 
-if TYPE_CHECKING:
-    from app.models.event import UserMessage
 
 
 class Persona(Base, table=True):
```

Comment:
here's a good example of the whole thing
- `fkey_` prefixed private columns for fkey cols
- `_id` suffixed getters and setters
- a relationship with the fkey_ primaryjoin defined